src/containers/Secrets/Secrets.test.js

```diff
@@ -127,9 +134,9 @@ it('click add new secret & modal appears', () => {
 
   jest.spyOn(API, 'getServiceAccounts').mockImplementation(() => []);
 
-  const { getByTestId } = render(
+  const { getByTestId } = renderWithRouter(
```

Comment:
we have a `renderWithIntl` helper so you don't need to manage this manually

Reply:
Ok, I have changed it to use ```renderWithIntl```